app/controllers/api/v1/participants_controller.rb

```diff
@@ -1,13 +1,35 @@
 module API
   module V1
     class ParticipantsController < BaseController
-      def index = head(:method_not_allowed)
+      include Pagination
+      include ::API::Concerns::FilterByUpdatedSince
+
+      def index
+        render json: to_json(paginate(participants_query.participants))
+      end
+
       def show = head(:method_not_allowed)
       def change_schedule = head(:method_not_allowed)
       def defer = head(:method_not_allowed)
       def withdraw = head(:method_not_allowed)
       def resume = head(:method_not_allowed)
       def outcomes = head(:method_not_allowed)
+
+    private
+
+      def participants_query
+        conditions = { lead_provider: current_lead_provider, updated_since: }
+
+        ::Participants::Query.new(**conditions.compact)
```

Comment:
There's no need to separate the params for `Query`, its clearer to just:
```ruby
::Participants::Query.new(lead_provider: current_lead_provider, updated_since:)
```

Reply:
Yeah I know I'm just trying to follow the pattern already being used in the other controllers.